### PR TITLE
R4iSDHC GOLD Pro/R4iSDHC RTS LITE/R4iSDHC Dual-Core 2021 are no longer support ntrboot

### PR DESCRIPTION
**Pull request by OasisAkari**
**Description**
I confirmed it from r4isdhc's official sale.
Since they replaced the mainboard used by the flashcard, the newer than 2020 model are no longer support ntrboot.

image

Translation (Use google translate):
```
Sorry for bothering you.

At present, the new R4 cards cost by ￥cny 20 on the market starting in 2021 do not support magnet flashing at all.

The cards that can be supported are part of the cards before 2020 and some high-priced cards.

None of the cost by ￥cny 20 or so cards newly released in 2021 support magnet flashing.

Must find the old version. So as not to increase our interpretation work.

Thank you. .
```

### CODE REVIEW

**Comment by hansoljin** [comment]
This needs to be looked further into. As far as I'm concerned, all carts from r4isdhc **.com** are supported. I haven't actually seen a 2021 **.com** cart yet (I doubt they're gonna release one at all), so it is yet to be seen.

However, it is definitely true that the r4isdhc **.com.cn** carts and the **.hk** carts don't support ntrboot (and never will.) 

Do you have an r4isdhc **.com** cart labeled 2021? If so, can you post a kernel? I could look into it a bit.

**Comment by OasisAkari** [comment]
> r4isdhc **.com** cart labeled 2021

ahh I can confirm r4isdhc **.com** cart labeled 2021(Available for official sales in mainland China (http://cn.r4isdhc.com/distributor/)) uses mainboard are replaced to r4isdhc.com.cn‘s one. So the kernel are same to r4isdhc.com.cn

image

**Comment by hansoljin** [comment]
Oh, I see. Then yes, it won't support ntrboot.

**Comment by R-YaTian** [comment]
> > r4isdhc **.com** cart labeled 2021
> 
> ahh I can confirm r4isdhc **.com** cart labeled 2021([Available for official sales in mainland China](http://cn.r4isdhc.com/distributor/)) uses mainboard are replaced to r4isdhc.com.cn's one. So the kernel are same to r4isdhc.com.cn
> 
> image
> 

The card www.r4isdhc.com belong to www.r4i-sdhc.com, this card on your photo is a fake card which used r4isdhc.com.cn's mainboard.
No need to add this commit.
And this fake card also support ntrboot, just choose 'R4iSDHC Dual-Core (r4isdhc.com.cn) family'.

**Comment by OasisAkari** [comment]
well, but I think this pr should leave in here for waiting for more people to check if so

**Comment by hansoljin** [comment]
> > r4isdhc **.com** cart labeled 2021
> 
> ahh I can confirm r4isdhc **.com** cart labeled 2021([Available for official sales in mainland China](http://cn.r4isdhc.com/distributor/)) uses mainboard are replaced to r4isdhc.com.cn's one. So the kernel are same to r4isdhc.com.cn
> 
> image
> 

I would wait until someone who has another r4isdhc **.com** cart to double check.

Also, we still don't have the proper flash backups for most of these, so I'd double check before saying all r4isdhc **.com.cn** ones will work with ntrboot.

**Comment by OasisAkari** [comment]
> The card [www.r4isdhc.com](http://www.r4isdhc.com) belongs to [www.r4i-sdhc.com](http://www.r4i-sdhc.com), this card on your photo is a fake card which used r4isdhc.com.cn's mainboard.
> No need to add this commit.
> And this fake card also support ntrboot, just choose 'R4iSDHC Dual-Core (r4isdhc.com.cn) family'.

I just tested it, nope choose that model option will also pop up an unsupported prompt

File: _pages/en_US/ntrboot.txt [hunk 1]
```diff
@@ -23,8 +23,8 @@ Note that carts with a "Time Bomb" will no longer be able to launch `.nds` files
...
 | [**R4i Gold 3DS**](http://www.nds-card.com/ProShow.asp?ProID=149) | $19.99 | No | <= 11.14.0 | <= 1.4.5 | All RTS revisions are compatible. |
 | [**R4i-SDHC 3DS RTS**](http://www.nds-card.com/ProShow.asp?ProID=146) | $13.99 | 1.85b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | |
-| [**R4iSDHC GOLD Pro 20XX**](http://www.nds-card.com/ProShow.asp?ProID=490) | $9.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013 models are incompatible. |
-| [**R4iSDHC RTS LITE 20XX**](http://www.nds-card.com/ProShow.asp?ProID=450) | $13.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013 models are incompatible. |
+| [**R4iSDHC GOLD Pro 20XX**](http://www.nds-card.com/ProShow.asp?ProID=490) | $9.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013, 2021 and later models are incompatible. |
+| [**R4iSDHC RTS LITE 20XX**](http://www.nds-card.com/ProShow.asp?ProID=450) | $13.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013, 2021 and later models are incompatible. |
```
Comment (lilyuwuu) [inline _pages/en_US/ntrboot.txt:27]: Use "Only cartridges marked with a year between 2014 and 2020 are compatible." Not sure how to make that concise and to clarify that it's inclusive of both 2014 and 2020, though...

File: _pages/en_US/ntrboot.txt [hunk 2]
```diff
@@ -23,8 +23,8 @@ Note that carts with a "Time Bomb" will no longer be able to launch `.nds` files
...
-| [**R4iSDHC GOLD Pro 20XX**](http://www.nds-card.com/ProShow.asp?ProID=490) | $9.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | Only r4isdhc **.com** carts are supported. All R4iSDHC 20XX flashcarts are identical. 2013 models are incompatible. |
-| [**R4iSDHC RTS LITE 20XX**](http://www.nds-card.com/ProShow.asp?ProID=450) | $13.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | Only r4isdhc **.com** carts are supported. All R4iSDHC 20XX flashcarts are identical. 2013 models are incompatible. |
+| [**R4iSDHC GOLD Pro 20XX**](http://www.nds-card.com/ProShow.asp?ProID=490) | $9.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | Only r4isdhc **.com** carts are supported. All R4iSDHC 20XX flashcarts are identical. Only cartridges marked with a year between 2014 and 2020 are compatible. |
+| [**R4iSDHC RTS LITE 20XX**](http://www.nds-card.com/ProShow.asp?ProID=450) | $13.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | Only r4isdhc **.com** carts are supported. All R4iSDHC 20XX flashcarts are identical. Only cartridges marked with a year between 2014 and 2020 are compatible. |
```
Comment (hansoljin) [inline _pages/en_US/ntrboot.txt:27]: If we wanted it simple, why not replace it all with something like:

`Only r4isdhc **.com** carts marked with a year between 2014 and 2020 are compatible.`

The R4iSDHC 20XX identical thing isn't really important if we make it clear specifically which ones work, anyway.

File: _pages/en_US/ntrboot.txt [hunk 3]
```diff
@@ -33,7 +33,7 @@ Note that carts with a "Time Bomb" will no longer be able to launch `.nds` files
 | **R4i Gold 3DS Deluxe "Starter"** | | No | 4.1.0 - 4.5.0 | <= 1.4.5 | |
 | **R4i Ultra** | | No | <= 4.3.0 | <= 1.4.5 | |
 | **R4i-SDHC 3DS RTS Deluxe Edition** | | Unknown | <= 11.14.0 | <= 1.4.5 | |
-| **R4iSDHC Dual-Core 20XX** | | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | Only r4isdhc **.com** carts are supported. All R4iSDHC 20XX flashcarts are identical.|
+| **R4iSDHC Dual-Core 20XX** | | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | Only r4isdhc **.com** carts are supported. All R4iSDHC 20XX flashcarts are identical. Only cartridges marked with a year between 2014 and 2020 are compatible. |
```
Comment (hansoljin) [inline _pages/en_US/ntrboot.txt:36]: same as above.

**Review by hansoljin — CHANGES_REQUESTED** [review]

**Comment by OasisAkari** [comment]
xd i'm stupid

**Comment by Rot-gut** [comment]
Does anyone have a backup .bin of the R4 Gold Pro 2021 card? I accidentally flashed R4 SDHC ntrboot onto it using a DSi and now it's a brick.

**Pull request merged**